src/biokbase/narrative/jobs/job.py

```diff
@@ -240,6 +238,76 @@ def state(self):
         except Exception as e:
             raise Exception(f"Unable to fetch info for job {self.job_id} - {e}")
 
+    def revised_state(self, state=None) -> dict:
+        """
+        :param state: can be queried individually from ee2/cache with self.state(),
+            but sometimes want it to be queried in bulk from ee2 upstream
+        :return: dict, with structure
+
+        {
+            owner: string (username, who started the job),
+            spec: app spec (optional)
+            widget_info: (if not finished, None, else...) job.get_viewer_params result
+            state: {
+                job_id: string,
+                status: string,
+                created: epoch ms,
+                updated: epoch ms,
+                queued: optional - epoch ms,
+                finished: optional - epoc ms,
+                terminated_code: optional - int,
+                tag: string (release, beta, dev),
+                parent_job_id: optional - string or null,
+                run_id: string,
+                cell_id: string,
+                errormsg: optional - string,
+                error (optional): {
+                    code: int,
+                    name: string,
+                    message: string (should be for the user to read),
+                    error: string, (likely a stacktrace)
+                },
+                error_code: optional - int
+            }
+        }
+        """
+        if not state:
+            state = self.state()
+        else:
+            state = self._augment_ee2_state(state)
+        state["child_jobs"] = []
+        batch_size = state.get("job_input", {}).get("narrative_cell_info", {}).get("batch_size")
+        if batch_size:
+            state["batch_size"] = batch_size
```

Comment:
This info has moved to the `extra_data` field; it's for the old style batch jobs -- the data is added in the `appmanager` module, but it doesn't get saved in ee2. I would be tempted just to get rid of this as it doesn't correlate with the info in `child_jobs` and caused me lots of confusion when I was working on the frontend.